### PR TITLE
Test: Make sure failure is propagated to next jobs

### PR DESCRIPTION
**Pull request by piratos**

### CODE REVIEW

File: tests/tests_e3/anod/test_sandbox.py [hunk 1]
```diff
@@ -433,3 +433,25 @@ def test_anodtest(git_specs_dir, e3fake_git_dir):
         conf_fd.write(git_repo_entry)
     p = e3.os.process.Run(command)
     assert 'svn vcs type not supported' in p.out
+
+
+def test_failure_status(git_specs_dir, e3fake_git_dir):
```
Comment (enzbang) [inline tests/tests_e3/anod/test_sandbox.py]: docstring missing here

File: tests/tests_e3/anod/test_sandbox.py [hunk 2]
```diff
@@ -433,3 +433,25 @@ def test_anodtest(git_specs_dir, e3fake_git_dir):
...
+        plan_fd.write("anod_test('e3fake')")
+    conf_dir = os.path.join(sandbox_dir, 'specs', 'conf.yaml')
+    git_repo_entry = E3Fake % 'no_url'
+    with open(conf_dir, 'w') as conf_fd:
+        conf_fd.write(git_repo_entry)
+    command = ['e3-sandbox', '-v', 'exec',
+               '--plan', plan_file,
+               sandbox_dir]
+    p = e3.os.process.Run(command)
+    assert p.out.count('status=  1') >= 6
```
Comment (enzbang) [inline tests/tests_e3/anod/test_sandbox.py]: why a space after the = sign? why 6? I have to say that I don't understand this test.

Comment (piratos) [inline tests/tests_e3/anod/test_sandbox.py]: the info logger has `status=%3d`

File: tests/tests_e3/anod/test_sandbox.py (hunk 1 above)
Comment (piratos) [inline tests/tests_e3/anod/test_sandbox.py]: Done

**Pull request merged**